requirements/requirements.txt

```diff
@@ -13,9 +13,9 @@ aiosignal==1.3.1
     # via aiohttp
 annotated-types==0.6.0
     # via pydantic
-    # via -r /awx_devel/requirements/requirements_git.txt
-ansiconv==1.0.0
+ansi2html==1.9.2
     # via -r /awx_devel/requirements/requirements.in
+    # via -r /awx_devel/requirements/requirements_git.txt
```

Comment:
Looks like this line was supposed to remain after line 15. Did you edit this file manually? It's supposed to be upgraded with a pip-tools run.

It seems like `requirements/updater.sh` is supposed to be used for this. Make sure to use the same version of Python and the same OS / CPU arch as the one used for the original generation.

Reply:
I used the updater to create the requirements file. I can run it again to make sure this is correct.

Reply:
That's weird. Perhaps the updater script has bugs, then.

Reply:
it runs in the container so it should always be the same @webknjaz it does seem odd that it moved that because neither of these I think are in our git requirements either.